canonical/ubuntu.com
# WD-11633 distributor checkout page

This PR needs to be merged first https://github.com/canonical/ubuntu.com/pull/13917
## Done
- Updated `/account/checkout` page for the distributor store
- Blocked by product listings (preview/purchase api is not working without product listing id)

## QA

1. Go to /pro/distributor
2. Click Initiate order
3. Complete all the steps /pro/distributor/shop (click "proceed to checkout" without updating anything in the /pro/distributor/shop to see VAT working)
4. Proceed to checkout
5. Click Checkout button in the /pro/distributor/order page
6.Check it goes `/account/checkout` page and all the details in the page

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-11633
